epub33/core/index.html

```diff
@@ -436,14 +453,23 @@ <h3>Terminology</h3>
 						<dfn id="dfn-path-name" data-lt="Path Names">Path Name</dfn>
 					</dt>
 					<dd>
-						<p>For a given directory within the <a href="#sec-container-abstract">OCF Abstract
-							Container</a>, the string holding all directory <a>File Name</a> in the full path
-							concatenated together with a <code>/</code> (<code>U+002F</code>) character separating the
-							directory File Names.</p>
-						<p>For a given file within the OCF Abstract Container, the Path Name is the string holding all
-							directory File Names concatenated together with a <code>/</code> character separating the
-							directory File Names, followed by a <code>/</code> character and then the File Name of the
-							file.</p>
+						<p>For a given <var>file</var> within the <a href="#sec-container-abstract">OCF Abstract
```

Comment:
Sometimes this refers to "files and directories" (eg in the terminology section) and sometimes just files. Are they really synonyms? This is particularly confusing below, where you talk about file names for all path components, some of which are directories. Maybe item or entity here, then specify directory below? Though naming arguments are always the best arguments.

Reply:
Per change proposed by @rdeltour this now says "file or directory"